docs/src/props.js

```diff
@@ -0,0 +1,77 @@
+import {Flash, Label} from '@primer/components'
+import {H3} from '@primer/gatsby-theme-doctocat/src/components/heading'
+import InlineCode from '@primer/gatsby-theme-doctocat/src/components/inline-code'
+import Paragraph from '@primer/gatsby-theme-doctocat/src/components/paragraph'
+import Table from '@primer/gatsby-theme-doctocat/src/components/table'
+import {graphql, useStaticQuery} from 'gatsby'
+import React from 'react'
+
+/** Render prop documentation for the given component */
+export function Props({of}) {
+  const data = useStaticQuery(graphql`
+    query {
+      allComponentMetadata {
```

Comment:
✍️ I am not very proficient with GraphQL, so I don’t know what makes `allComponentMetadata` queryable, but quick searches will show other examples of this pattern, e.g. https://blog-nrrd.doi.gov/implementing-mdx/#Step-1-Using-Gatsbys-GraphQL-to-get-your-components